src/bot.js

```diff
@@ -40,10 +43,14 @@ client.on('message', (message) => {
       case 'l':
       case 'list':
```

Comment:
Looks like you need to update this so it lists both the Verdansk locations & Rebirth Island locations.